src/covid19model/visualization/optimization.py

```diff
@@ -0,0 +1,146 @@
+import datetime
+import random
+import matplotlib
+import matplotlib.pyplot as plt
+import matplotlib.dates as mdates
+import pandas as pd
+import numpy as np
+
+
+# From Color Universal Design (CUD): https://jfly.uni-koeln.de/color/
+colorscale_okabe_ito = {"orange" : "#E69F00", "light_blue" : "#56B4E9",
+                        "green" : "#009E73", "yellow" : "#F0E442",
+                        "blue" : "#0072B2", "red" : "#D55E00",
+                        "pink" : "#CC79A7", "black" : "#000000"}
+
+# covid 19 specific parameters
+plt.rcParams.update({
+    "axes.prop_cycle": plt.cycler('color',
+                                  list(colorscale_okabe_ito.values())),
+    "font.size": 15,
+    "lines.linewidth" : 3,
+    "axes.spines.top": False,
+    "axes.spines.right": False,
+    "ytick.major.left": True,
+    "axes.grid": True
+})
+
+
+def _apply_tick_locator(ax):
+    """support function to apply default ticklocator settings"""
+    ax.xaxis.set_major_locator(plt.MaxNLocator(5))
+    ax.yaxis.set_major_locator(plt.MaxNLocator(4))
+    return ax
+
+def plot_fit(model,data,startDate,states,checkpoints=None,samples=None,filename=None,dataMkr=['o','v','s','*','^'],
+            modelClr=['green','orange','red','black','blue'],legendText=None,titleText=None,getfig=False,ax=None):
```

Comment:
In python it is a convention to use lowercase variable names, Hence, `startDate` -> `start_date`,.... (see https://www.python.org/dev/peps/pep-0008/#naming-conventions)  It is also a convention to add a space after the comma.

Reply:
resolved.